web/app.fluidity.money/app/routes/wtf.tsx

```diff
@@ -90,6 +90,10 @@ export const loader: LoaderFunction = async () => {
   });
 };
 
+export const headers: HeadersFunction = () => ({
+  "Cache-Control": "public, s-maxage=60",
+})
+
```

Comment:
I'm going to have a look at the loader to see whether this makes sense.

Reply:
Not ideal as this will cache error responses. It doesn't allow client invalidation either.
I'd say convert the loader here to a resource route and consider using the `useCache` hook.

Alternatively, a useFetcher and just wait.
https://remix.run/docs/en/v1/api/remix#fetcherload
https://remix.run/docs/en/v1/guides/api-routes#resource-routes (For a useFetcher brief, scroll up)